sdk/avalon_sdk/connector/direct/jrpc/jrpc_worker_registry.py

```diff
@@ -94,7 +98,15 @@ def worker_lookup(self, worker_type=None, organization_id=None,
         }
 
         if worker_type is not None:
-            json_rpc_request["params"]["workerType"] = worker_type.value
+            if isinstance(worker_type, WorkerType):
+                json_rpc_request["params"]["workerType"] = worker_type.value
+            elif isinstance(worker_type, int):
```

Comment:
We need to check if worker_type falls in the range of 1 to 4, if its integer.

Reply:
done